recipes/knotinframe/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "2.2.14" %}
+{% set sha256 = "da7371378e32b2329da226be79a11974fdc4e66fe0b664757b5aafe29e2e12d6" %}
+
+package:
+  name: knotinframe
+  version: {{ version }}
+
+source:
+  sha256: {{ sha256 }}
+  url: https://github.com/jlab/fold-grammars/archive/{{ version }}.tar.gz
+
+build:
+  number: 0
+  skip: False # [osx]
```

Comment:
This doesn’t do anything because `skip` is False by default. If you want to skip building the recipe on macOS, you need to write this as `skip: True  # [osx]` (yes, it’s confusing).